package.json

```diff
@@ -76,6 +76,6 @@
         "swc-loader": "^0.2.3",
         "ts-jest": "^28.0.5",
         "ts-node": "^10.8.1",
-        "typescript": "^4.8.2"
+        "typescript": "^4.9.5"
```

Comment:
Bump typescript. In #3833 it was slightly convenient to have the [`satisfies` operator](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-9.html#the-satisfies-operator, which as included in TS 4.9.

Corresponding bumps to swc and prettier to allow using `satisfies`.